client/src/FindSchool.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { Button, ButtonGroup } from "@chakra-ui/react";
+
+function FindSchool () {
+    return (
+        <div>
+        <div className="schoolForm">
+
+            <SearchElement text="Country " />
+            <SearchElement text="Location " />
+            <SearchElement text="School Name " />
+            <SearchElement text="Responsible name " />
+            <SearchElement text="When deployed " />
+            <SearchElement text="Status " />
+        </div>
+        <AddButton />
+        </div>
+    );
+}
+
+function SearchElement (props) {
+    return (
+        <div>
+            <span>{props.text}</span>
+            <input className="inputBox" type="text" />
+        </div>
```

Comment:
Here we can use form element from Chakra UI
```suggestion
        <div>
            <FormLabel htmlFor={props.text}>{props.text}</span>
            <Input id={props.text} className="inputBox" type="text" />
        </div>
```

For group of elements, we can use `<FormControl></FormControl>` which is same as using `<InputGroup>` tag in HTML

Reply:
Yes, I changed the code